.github/workflows/ci.yml

```diff
@@ -68,9 +65,20 @@ jobs:
         env:
           RUST_BACKTRACE: short
 
-  db_migration:
-    name: "Database Migration"
-    runs-on: ubuntu-22.04-16core
+  protobuf_backward_compat:
+    name: "Protobuf Backward Compatibility"
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - uses: bufbuild/buf-setup-action@1158f4fa81bc02e1ff62abcca6d516c9e24c77da
+      - uses: bufbuild/buf-breaking-action@a074e988ee34efcd4927079e79c611f428354c01
+        with:
+          against: "https://github.com/near/nearcore.git#${{github.event.pull_request.base.sha && format('ref={0}', github.event.pull_request.base.sha) || 'branch=master' }}"
```

Comment:
I assume these lines were not changed, and are displayed this way by github only due to reordering